src/WebJobs.Script/Utility.cs

```diff
@@ -33,6 +33,11 @@ public static string GetSubscriptionId()
             return null;
         }
 
+        public static bool IsDynamicSku()
```

Comment:
Move this to the `ScriptSettingsManager` as a property (similar to `IsAzureEnvironment`, `IsRemoteDebuggingEnabled`)